editor/editor_node.cpp

```diff
@@ -225,6 +227,10 @@ void EditorNode::_unhandled_input(const Ref<InputEvent> &p_event) {
 				_bottom_panel_switch(false, i);
 			}
 		}
+
+		if (old_editor != editor_plugin_screen) {
+			k->set_pressed(false);
```

Comment:
Wouldn't it work with `get_tree()->set_input_as_handled();`? That would be cleaner.

Reply:
Or @hpvb mentioned `accept_event()`. Not sure which would work, if any.

Reply:
accept_event() won't work here as EditorNode is not a control :-/